### PR TITLE
[docs] fix misc. grammar and punctuation.

### PR DESCRIPTION
**Pull request by JesterOrNot**
Fix a few grammar mistakes.

### CODE REVIEW

File: src/docs/languages/html.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gitpod comes with [emmet](http://emmet.io/) support right out of the box giving
 ## Live Preview
 
 ![image of preview](../images/live_preview.png)
-With Gitpod you can open a preview for Markdown and HTML files while you are coding in the upper right hand corner ![preview icon](../images/preview.png) click it see the result in real time!
+With Gitpod you can open a preview for Markdown and HTML files while you are coding in the upper right-hand corner ![preview icon](../images/preview.png) click it see the result in real-time!
```
Comment (nisarhassan12) [inline src/docs/languages/html.md]: Do we really need dashes here :slightly_smiling_face: ?

Comment (JesterOrNot) [reply src/docs/languages/html.md]: Grammarly says so :smiley:

File: src/docs/languages/rust.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Welcome, Rustacean!
 
-Rust is a first-class language in Gitpod, and we believe that Gitpod is a great tool for Rust projects. In fact, some of Gitpod's core developers frequently work with Rust code (in Gitpod), and several key components of Gitpod are written in Rust:
```
Comment (nisarhassan12) [inline src/docs/languages/rust.md]: Thanks for this change but I would still like to keep  "in fact, some" Are there any grammatical reasons behind the change ?

File: src/docs/languages/html.md (hunk 1 above)
Comment (nisarhassan12) [reply src/docs/languages/html.md]: Ok! Thanks! I don't know Grammar Well :laughing:

File: src/docs/languages/rust.md (hunk 2 above)
Comment (JesterOrNot) [reply src/docs/languages/rust.md]: Grammarly said it was just un-needed I will undo this

File: src/docs/languages/html.md (hunk 1 above)
Comment (JesterOrNot) [reply src/docs/languages/html.md]: Sadly, neither do I. lol

File: src/docs/languages/rust.md (hunk 2 above)
Comment (nisarhassan12) [reply src/docs/languages/rust.md]: Ok! then let's go for it.

Comment (JesterOrNot) [reply src/docs/languages/rust.md]: OK

**Pull request merged**